src/renderer/components/log-time-view.tsx

```diff
@@ -0,0 +1,133 @@
+import { SleuthState } from '../state/sleuth';
+import { observer } from 'mobx-react';
+import React from 'react';
+import { ChartJSChart, InteractionItem } from './chart-js';
+import { parse } from 'date-fns';
+import zoomPlugin from 'chartjs-plugin-zoom';
+import { isLogFile } from '../../utils/is-logfile';
+import { LogLevel, LogMetrics } from '../../interfaces';
+import autoBind from 'react-autobind';
+
+export interface LogTimeViewProps {
+  state: SleuthState;
+  isVisible: boolean;
+  height: number;
+}
+
+@observer
+export class LogTimeView extends React.Component<LogTimeViewProps> {
+  constructor(props: LogTimeViewProps) {
+    super(props);
+    autoBind(this);
+  }
+
+  private onChartClick(chartElements: Array<InteractionItem>) {
+    if (!chartElements || !chartElements.length) return;
```

Comment:
Either we don't need the `!chartElements` on the type of `chartElements` is missing a `| null`